### PR TITLE
[WFLY-18666] Fix i18n of exception messages in weld

### PR DESCRIPTION
**Pull request by soul2zimate**
Issue: https://issues.redhat.com/browse/WFLY-18666

[WFLY-18666] Fix i18n of exception messages in weld

### CODE REVIEW

File: weld/common/src/main/java/org/jboss/as/weld/logging/WeldLogger.java [hunk 1]
```diff
@@ -249,4 +251,19 @@ public interface WeldLogger extends BasicLogger {
...
     @Message(id = 58, value = "Persistence unit '%s' removed.")
     IllegalStateException persistenceUnitRemoved(String scopedPuName);
+
+    @Message(id = 59, value = "Unknown deployment type %s")
+    IllegalArgumentException unknownDeploymentType(DeploymentType deploymentType);
+
+    @Message(id = 60, value = "Error handling file %s")
+    RuntimeException errorHandlingFile(File file, @Cause Throwable throwable);
+
+    @Message(id = 61, value = "\"Exactly one service provider is required for: %s")
```
Comment (manovotn) [inline weld/common/src/main/java/org/jboss/as/weld/logging/WeldLogger.java:261]:
```suggestion
    @Message(id = 61, value = "Exactly one service provider is required for: %s")
```

**Review by manovotn — APPROVED** [review]
Added one small comment, otherwise LGTM.

File: weld/common/src/main/java/org/jboss/as/weld/logging/WeldLogger.java (hunk 1 above)
Comment (soul2zimate) [reply weld/common/src/main/java/org/jboss/as/weld/logging/WeldLogger.java]: oops, fixed. Thanks

**Comment by bstansberry** [comment]
Thanks @soul2zimate and @manovotn

**Pull request merged**